src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+import Display from './Display';
+import ButtonPannel from './ButtonPannel';
+
+function App() {
+  return (
+    <>
+      <Display result={0} />
```

Comment:
- [ ] <b>OPTIONAL: </b> There is no need to initialize default value for result in here since you already did so in Display component in `Display.defaultProps`.